libs/feature/map/src/lib/map-context/map-context.service.ts

```diff
@@ -112,13 +116,22 @@ export class MapContextService {
           }),
         })
       case MapContextLayerTypeEnum.WMS:
-        return new TileLayer({
-          source: new TileWMS({
-            url: layerModel.url,
-            params: { LAYERS: layerModel.name },
-            gutter: 20,
-          }),
-        })
+        if (mapConfig?.DO_NOT_TILE_WMS) {
+          return new ImageLayer({
+            source: new ImageWMS({
+              url: layerModel.url,
+              params: { LAYERS: layerModel.name },
+            }),
+          })
+        } else {
+          return new TileLayer({
+            source: new TileWMS({
+              url: layerModel.url,
+              params: { LAYERS: layerModel.name },
```

Comment:
to let backend components know that we're querying tiles we should also set the `TILED: true` parameter
See e.g. https://docs.geoserver.org/main/en/user/geowebcache/using.html